server/plugin/command.go

```diff
@@ -132,7 +132,7 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 		}
 
 		var client Client
-		if action != constants.CommandDisconnect && action != constants.CommandSearchAndShare {
+		if action != constants.CommandDisconnect {
```

Comment:
by removing this condition we are also checking if the subscriptions are configured for a user or not (at line 140) for the search and share command, which is not needed, so we should update this